src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.io.ByteArrayOutputStream;
+import java.io.FileDescriptor;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+/**
+ * Represents a Duke class which is used to run a taskList computer program
+ * that allows users to store their various task (Deadline, Event, Things to do)
+ * and store them in a text file. The program also allows them to delete and update
+ * their list once they have completed it.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for Duke class
+     * instantiate a UI, Storage and TaskList object.
+     *
+     */
+    public Duke() {
+        ui = new Ui();
+        storage = new Storage("D:\\NUS Computer Science\\CS2103T\\duke\\src\\main\\java\\duke.txt");
```

Comment:
Prefer relative path over full path. Full path are hardly usable in deployments.
```suggestion
        storage = new Storage("duke.txt");
```